uber/templates/regform.html

```diff
@@ -96,12 +96,17 @@
         <div class="col-sm-6">
             <input type="text" name="last_name" id="last_name" value="{{ attendee.last_name }}" class="form-control" placeholder="Last Name">
         </div>
-        {% if c.PAGE_PATH != '/registration/form' %}
-            <p class="help-block col-sm-6 col-sm-offset-2">
-                These must be your legal name.
-                <span class="popup">{% popup_link "../static_views/legal_name.html" "What does this mean?" %}</span>
-            </p>
-        {% endif %}
+    </div>
+
+    <div class="form-group">
```

Comment:
I've been thinking about this a lot. There are two problems we have with the current way we handle legal names:

1. People can't have separate preferred names and legal names (what this PR is definitely going to solve)
2. People consistently ignore our pleas to fill out their legal name as it appears on their Photo ID, leading to problems down the road when volunteers try to search for them.

I think your solution is a good way to handle solving the first problem. But if we CAN and if people don't think it's too onerous, I'd rather flip the logic of that checkbox to say, "This is my name exactly as it appears on my photo ID." Then:

1. If people don't check that checkbox AND don't fill out 'legal name' we use a form validation to kick them back, e.g., "You must either confirm that your name appears exactly this way on your photo ID or enter your legal name separately"
2. If someone checks the checkbox we can first clear, then set `disabled` on the legal_name field, making it clear that it's not used

The only annoying part is that this might require a new field in the DB. We could probably fake recreating the checkbox's state based on the state of legal_name, though.